### ISSUE TITLE
docs/.vuepress/styles/index.styl load error

### ISSUE THREAD

**Issue by zq99299** [vuejs/vuepress#1170]
- [x ] I confirm that this is a issue rather than a question.

#### Version 
1.0.0-alpha.30

#### Steps to reproduce

1.  https://github.com/zq99299/note-book
2.  yarn install
3.  yarn docs:dev

docs/.vuepress/styles/index.styl
```
.content {
  font-size 30px;
}
```

#### What is expected?

Function is normal, style is applied normally

#### What is actually happening?

```
[./node_modules/webpack-hot-client/client/log.js] (webpack)-hot-client/client/log.js 2.4 KiB {app} [built]
[./node_modules/webpack-hot-client/client/socket.js] (webpack)-hot-client/client/socket.js 1.39 KiB {app} [built]
    + 893 hidden modules

ERROR in ./node_modules/@vuepress/core/.temp/style.styl (./node_modules/css-loader??ref--13-oneOf-1-1!./node_modules/postcss-loader/lib??ref--13-oneOf-1-2!./node_modules/stylus-loader??ref--13-oneOf-1-3!./node_modules/@vuepress/core/.temp/style.styl)
Module build failed (from ./node_modules/stylus-loader/index.js):
Error: H:\dev\note\note-book\node_modules\@vuepress\core\.temp\style.styl:1:72
   1| @import("H:\\dev\\note\\note-book\\docs\\.vuepress\\styles\\index.styl")
-----------------------------------------------------------------------------^

failed to locate @import file H:\\dev\
ote\
ote-book\\docs\\.vuepress\\styles\\index.styl

    at CachedPathEvaluator.visitImport (H:\dev\note\note-book\node_modules\stylus-loader\lib\evaluator.js:157:21)
    at CachedPathEvaluator.Visitor.visit (H:\dev\note\note-book\node_modules\stylus\lib\visitor\index.js:28:40)
    at CachedPathEvaluator.Evaluator.visit (H:\dev\note\note-book\node_modules\stylus\lib\visitor\evaluator.js:160:18)
    at CachedPathEvaluator.Evaluator.visitRoot (H:\dev\note\note-book\node_modules\stylus\lib\visitor\evaluator.js:707:27)
    at CachedPathEvaluator.Visitor.visit (H:\dev\note\note-book\node_modules\stylus\lib\visitor\index.js:28:40)
    at CachedPathEvaluator.Evaluator.visit (H:\dev\note\note-book\node_modules\stylus\lib\visitor\evaluator.js:160:18)
    at CachedPathEvaluator.Evaluator.evaluate (H:\dev\note\note-book\node_modules\stylus\lib\visitor\evaluator.js:247:15)
    at Renderer.render (H:\dev\note\note-book\node_modules\stylus\lib\renderer.js:86:26)
    at H:\dev\note\note-book\node_modules\stylus-loader\index.js:167:12
    at tryCatchReject (H:\dev\note\note-book\node_modules\when\lib\makePromise.js:840:30)
    at runContinuation1 (H:\dev\note\note-book\node_modules\when\lib\makePromise.js:799:4)
    at Fulfilled.when (H:\dev\note\note-book\node_modules\when\lib\makePromise.js:590:4)
    at Pending.run (H:\dev\note\note-book\node_modules\when\lib\makePromise.js:481:13)
    at Scheduler._drain (H:\dev\note\note-book\node_modules\when\lib\Scheduler.js:62:19)
    at Scheduler.drain (H:\dev\note\note-book\node_modules\when\lib\Scheduler.js:27:9)
    at _combinedTickCallback (internal/process/next_tick.js:132:7)
 @ ./node_modules/@vuepress/core/.temp/style.styl 4:14-194 14:3-18:5 15:22-202
 @ ./node_modules/@vuepress/core/lib/internal-plugins/style/client.js
 @ ./node_modules/@vuepress/core/.temp/app-enhancers/0.js
 @ ./node_modules/@vuepress/core/.temp/internal/app-enhancers.js
 @ ./node_modules/@vuepress/core/lib/app/app.js
 @ ./node_modules/@vuepress/core/lib/app/clientEntry.js
 @ multi ./node_modules/@vuepress/core/lib/app/clientEntry.js
Child html-webpack-plugin for "index.html":
         Asset      Size  Chunks  Chunk Names
    index.html  1.38 MiB       0
    Entrypoint undefined = index.html
    [./node_modules/lodash/lodash.js] 527 KiB {0} [built]
    [./node_modules/vuepress-html-webpack-plugin/lib/loader.js!./node_modules/@vuepress/core/lib/app/index.dev.html] 424 bytes {0} [built]
    [./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 497 bytes {0} [built]
```

#### Other relevant information

- Your OS: windows 10
- Node.js version: v8.12.0
- Browser version: Google Chrome 版本 69.0.3497.81（正式版本） （64 位）
- Is this a global or local install?  local
- Which package manager did you use for the install? yarn

**Comment by zq99299** [comment]
@meteorlxy It looks like it should be. Has this version been released yet?

**Comment by zq99299** [comment]
Under 1.0.0-alpha.31

```
[./node_modules/webpack-hot-client/client/socket.js] (webpack)-hot-client/client/socket.js 1.39 KiB {app}
    + 930 hidden modules

WARNING in ./docs/elasticsearch/core/index.md
Module Warning (from ./node_modules/@vuepress/markdown-loader/index.js):

File for relative link "core/search-engine/44-mapping-detailed.md" does not exist.
(Resolved file: H:\dev\note\note-book\docs\elasticsearch\core\core\search-engine\44-mapping-detailed.md)

 @ ./node_modules/@vuepress/core/.temp/internal/page-components.js 43:22-93
 @ ./node_modules/@vuepress/core/lib/app/util.js
 @ ./node_modules/@vuepress/core/lib/app/components/Content.js
 @ ./node_modules/@vuepress/core/lib/app/app.js
 @ ./node_modules/@vuepress/core/lib/app/clientEntry.js
 @ multi ./node_modules/@vuepress/core/lib/app/clientEntry.js

WARNING in ./docs/elasticsearch/core/index.md
Module Warning (from ./node_modules/@vuepress/markdown-loader/index.js):

File for relative link "core/search-engine/44-mapping-detailed.md" does not exist.
(Resolved file: H:\dev\note\note-book\docs\elasticsearch\core\core\search-engine\44-mapping-detailed.md)

 @ ./node_modules/@vuepress/core/.temp/internal/page-components.js 43:22-93
 @ ./node_modules/@vuepress/core/lib/app/util.js
 @ ./node_modules/@vuepress/core/lib/app/components/Content.js
 @ ./node_modules/@vuepress/core/lib/app/app.js
 @ ./node_modules/@vuepress/core/lib/app/clientEntry.js
 @ multi ./node_modules/@vuepress/core/lib/app/clientEntry.js

WARNING in ./docs/elasticsearch/core/index.md
Module Warning (from ./node_modules/@vuepress/markdown-loader/index.js):

File for relative link "core/search-engine/44-mapping-detailed.md" does not exist.
(Resolved file: H:\dev\note\note-book\docs\elasticsearch\core\core\search-engine\44-mapping-detailed.md)

 @ ./node_modules/@vuepress/core/.temp/internal/page-components.js 43:22-93
 @ ./node_modules/@vuepress/core/lib/app/util.js
 @ ./node_modules/@vuepress/core/lib/app/components/Content.js
 @ ./node_modules/@vuepress/core/lib/app/app.js
 @ ./node_modules/@vuepress/core/lib/app/clientEntry.js
 @ multi ./node_modules/@vuepress/core/lib/app/clientEntry.js

ERROR in ./node_modules/@vuepress/theme-default/components/Home.vue?vue&type=template&id=748f7afd& (./node_modules/cache-loader/dist/cjs.js?{"cacheDirectory":"node_modules//@vuepress//core//node_modules//.cache//vuepress","cacheIdentifier":"22d00c0c-vue-loader-template"}!./node_modules/vue-loader/lib/loaders/templateLoader.js??vue-loader-options!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./node_modules/@vuepress/theme-default/components/Home.vue?vue&type=template&id=748f7afd&)
Module Error (from ./node_modules/vue-loader/lib/loaders/templateLoader.js):
(Emitted value instead of an instance of Error)
  Error compiling template:

  <main class="home" aria-labelledby="main-title">
    <header class="hero">
      <img
        v-if="data.heroImage"
        :src="$withBase(data.heroImage)"
        alt="hero"
      >

      <h1 id="main-title">{{ data.heroText || $title || 'Hello' }}</h1>

      <p class="description">
        {{ data.tagline || $description || 'Welcome to your VuePress site' }}
      </p>

      <p
        class="action"
        v-if="data.actionText && data.actionLink"
      >
        <NavLink
          class="action-button"
          :item="actionLink"
        />
      </p>
    </header>

    <div
      class="features"
      v-if="data.features && data.features.length"
    >
      <div
        class="feature"
        v-for="(feature, index) in data.features"
        :key="index"
      >
        <h2>{{ feature.title }}</h2>
        <p>{{ feature.details }}</p>
      </div>
    </div>

    <Content class="custom"/>

    <div
      class="footer"
      v-if="data.footer"
    >
      {{ data.footer }}
    </div>
  </div>

  - tag <main> has no matching end tag.

 @ ./node_modules/@vuepress/theme-default/components/Home.vue?vue&type=template&id=748f7afd& 1:0-389 1:0-389
 @ ./node_modules/@vuepress/theme-default/components/Home.vue
 @ ./node_modules/cache-loader/dist/cjs.js??ref--3-0!./node_modules/babel-loader/lib??ref--3-1!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader/lib??vue-loader-options!./node_modules/@vuepress/theme-default/layouts/Layout.vue?vue&type=script&lang=js&
 @ ./node_modules/@vuepress/theme-default/layouts/Layout.vue?vue&type=script&lang=js&
 @ ./node_modules/@vuepress/theme-default/layouts/Layout.vue
 @ ./node_modules/@vuepress/core/.temp/internal/layout-components.js
 @ ./node_modules/@vuepress/core/lib/app/util.js
 @ ./node_modules/@vuepress/core/lib/app/components/Content.js
 @ ./node_modules/@vuepress/core/lib/app/app.js
 @ ./node_modules/@vuepress/core/lib/app/clientEntry.js
 @ multi ./node_modules/@vuepress/core/lib/app/clientEntry.js
Child html-webpack-plugin for "index.html":
                                   Asset      Size  Chunks             Chunk Names
    19007765df46148032aa.hot-update.json  44 bytes          [emitted]
                              index.html  1.38 MiB       0
    Entrypoint undefined = index.html
    [./node_modules/lodash/lodash.js] 527 KiB {0}
    [./node_modules/vuepress-html-webpack-plugin/lib/loader.js!./node_modules/@vuepress/core/lib/app/index.dev.html] 424 bytes {0}
    [./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 497 bytes {0}

```

**Comment by meteorlxy** [comment]
@zq99299 It has been fixed by #1181 , but no newer version released.

Fix the version of `@vuepress/theme-default` to `1.0.0-alpha.30`

**Comment by zq99299** [comment]
1.0.0-alpha.32 Have to solve
